style/modules/scrollCarousel.css

```diff
@@ -0,0 +1,24 @@
+.scrollCarousel {
+    max-height: 300px;
+    overflow: hidden;
+}
+
+.scrollCarousel--contents {
+    width: max-content;
+    display: flex;
+    animation-direction: alternate;
+    animation-name: moveSlideshow;
+    animation-duration: 5s;
```

Comment:
we'll probably want `animation-duration` to be at least `20s` -- not sure of the exact value, but you should take a look at the missionbit.org homepage and get it roughly that kind of speed.